controllers/spotify.controller.js

```diff
@@ -0,0 +1,48 @@
+
+const spotifyApi = require('../models/spotify');
+
+module.exports.home = (req, res) => {
+    res.render('home')
+}
+
+module.exports.searchArtist = (req, res) => {
+    
```

Comment:
podemos pasar el req.query.name a una variable para que sea mas facil ara leer al utilizarla 

```suggestion
const artistName = req.query.name
    spotifyApi
        .searchArtists(artistName)
```